### PR TITLE
Mf  create joins

### PR DESCRIPTION
**Pull request by algorithmike**
created joins and association classes to close issue #68

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 from db import query
 
 
+temp = query.get_all_employees()
```
Comment (eliasargandara) [inline app.py]: @algorithmike Once you make a pull request, you should remove any code you were only using for testing.

File: db/models.py [hunk 2]
```diff
@@ -1,5 +1,8 @@
-from sqlalchemy import Column, Integer, String
+from sqlalchemy import Column, Integer, String, ForeignKey, Table
 from db.database import Base
+from sqlalchemy.orm import sessionmaker, relationship
```
Comment (eliasargandara) [inline db/models.py]: @algorithmike There should not be any imports in a file that are not being used.

Comment (eliasargandara) [reply db/models.py]: The `sessionmaker `import does not seem to be used in the rest of the file.

File: npm-debug.log [hunk 3]
```diff
@@ -0,0 +1,23 @@
+0 info it worked if it ends with ok
```
Comment (eliasargandara) [inline npm-debug.log]: @algorithmike We should add log files to the .gitignore file.

File: db/models.py [hunk 4]
```diff
@@ -98,6 +99,9 @@ def __repr__(self):
         values = (self.app_id, self.name)
         return str_format % values
         
+    def toJSON(self):
```
Comment (eliasargandara) [inline db/models.py]: @algorithmike If I remember correctly, lines 102-105 do not work. They should be removed.

**Review by eliasargandara — CHANGES_REQUESTED** [review]

File: npm-debug.log (hunk 3 above)
Comment (algorithmike) [reply npm-debug.log]: Ok to delete npm-debug.log from commit

**Review by eliasargandara — APPROVED** [review]

**Pull request merged**